tools/metrics/METADATA

```diff
@@ -0,0 +1,17 @@
+name: "metrics"
+description:
+  "Filtered subtree at tools/metrics."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://github.com/chromium/chromium"
+  }
+  # Closest commit hash to m70.
+  version: "29f3ad3470f20c7aa50ac6ab01809311a8c0db87"
```

Comment:
The reference based on this `METADATA` file seems not to exist:
https://github.com/chromium/chromium/commit/29f3ad3470f20c7aa50ac6ab01809311a8c0db87

I think we should use a release tag here, probably `70.0.3538.124` in this case,
and get the tree from there so it matches what's in Chromium:
https://chromium.googlesource.com/chromium/src/+/70.0.3538.124/tools/metrics

Reply:
Fixed the commit hash.

How did you arrive on the `70.0.3538.124` tag? Based off the files in this commit? The target hash (matching other metric repos brought in) was `734293972a4b3f6d949e489ac02145538fb59c70` in https://chromium.googlesource.com/chromium/src/components/metrics/ which translated to `fd3a73980b82eea000403c2c9c1f204f6612e7cf` in https://github.com/chromium/chromium and then filtering the repo turned it into what's there now with `05c10502d7ddf42b0d1624eb821ca41fd7a8bef9`

Reply:
> How did you arrive on the `70.0.3538.124` tag?

I just found the most recent release tag for Chromium 70
based on the comment `# Closest commit hash to m70.`
Incidentally I meant to ask out of curiosity: how did you pick 70?

Reply:
The closest commit hash to m70 was probably one of the original justifications for using `734293972a4b3f6d949e489ac02145538fb59c70` and now the goal is whatever is closest to `734293972a4b3f6d949e489ac02145538fb59c70` to keep it consistent with the other repos. Joel might have more context on why that specific hash was chosen as the target?

Reply:
IIRC, when we checked in //components/metrics, I looked around at the Chromium base libraries in Cobalt, and it seemed M70 was generally the last time we synced. That's why it was chosen.

In this case, //tools/metrics is primarily used as a catalog of histograms in the codebase. It's both the documentation for the histograms as well as a mechanism to "unhash" the histogram names after logging to the server. What's important is that the histogram xml files in //tools/metrics generally be a superset of the histograms we log in Cobalt.

I'm realizing this is likely tricky as histograms are all over the codebase and I'm not sure the likelihood of all them being "in sync" with a single Chromium version. Is M70 generally the basis for most of our Chromium code? If so, I'd say, let's just start with this. If there are metrics not included, we can update this directory later and it shouldn't be too big of  a deal. This isn't code that's used in the Cobalt binary, it's more just histogram metadata management. So we'll usually be working from trunk anyway.

Reply:
Just checking on what's blocking this PR. It's blocking me making next steps on the google internal telemetry integration. What's the ask here? Just link this to a tag instead? SGTM.

Reply:
I'm thinking we can just import the latest version of this package from Chromium instead of something around M70... AFAICT, this should still work well for my purposes. In fact, it might even just be strictly better.

Reply:
Maybe we should use the latest LTS release:
https://chromium.googlesource.com/chromium/src/+/114.0.5735.284/tools/metrics

Reply:
+1 SGTM

Reply:
See related (perhaps not strictly necessary) pull #1068.

Reply:
I don't think jumping all the way to 114 is the right call here. It's way more recent than what we have in other metrics related layers. If this is destined for 24LTS it's way too late to aim for that.

Older version that doesn't need us bumping language requirements is fine

Reply:
The older metrics code required bumping (?) to Python 2, as noted above.
No revision is compatible with Python 3.7.